### ISSUE TITLE
Handle missing tokenizer_class

### ISSUE THREAD

**Issue by baloodevil** [xenova/transformers.js#245]
tokenizers.js expects `tokenizer_class` to exist in a `tokenizer_config.json` file according to [this line](https://github.com/xenova/transformers.js/blob/46dd49064f681d017e5ff71a75c0b96e2d5686be/src/tokenizers.js#L3533).  But a common model like [`bert-base-uncased`](https://huggingface.co/bert-base-uncased/resolve/main/tokenizer_config.json) does not have `tokenizer_class`.  So the following code 
```
import { AutoTokenizer } from "@xenova/transformers";
const tokenizer = await AutoTokenizer.from_pretrained("bert-base-uncased");
const { input_ids } = await tokenizer("I love transformers!");
```

throws `error - TypeError: Cannot read properties of undefined (reading 'replace')`

**Comment by xenova** [comment]
Hi there. Please use `Xenova/bert-base-uncased` as it provides an up-to-date tokenizer_config.json

**Comment by xenova** [comment]
> The documentation [page](https://huggingface.co/docs/transformers.js/api/tokenizers) should be updated with that model name and add information on what models will work (if it doesn't exist elsewhere).

Oh yes, good spot! Feel free to open a PR if you'd like (or I can do it when I have some time). All that needs to be changed is this:
https://github.com/xenova/transformers.js/blob/46dd49064f681d017e5ff71a75c0b96e2d5686be/src/tokenizers.js#L10

Do get your model working, you can use our conversion tool: https://huggingface.co/docs/transformers.js/custom_usage#convert-your-models-to-onnx

1. Clone the repo
2. Install the requirements:
    ```
    pip install -r scripts/requirements.txt
    ```
3. Run the conversion command, replacing with your model ID
    ```
    python -m scripts.convert --quantize --model_id bert-base-uncased
    ```

**Comment by baloodevil** [comment]
Cloned and made doc changes, but forking your repo then pushing my changes gives error...
```
> git push -u origin main
To https://github.com/baloodevil/transformers.git
 ! [rejected]        main -> main (fetch first)
error: failed to push some refs to 'https://github.com/baloodevil/transformers.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```
I tried pulling first.  If you know the solution to the git process, great, but if not, probably not worth your time to resolve just to get lines of docs updated.

Working on converting to ONNX now.  It seems to be stopped at `Ignore MatMul due to non constant B: /[/distilbert/transformer/layer.5/attention/MatMul_1]`.  I may need to update cuda from 11.6 or install TensorRT and cuBLAS if [this](https://github.com/ELS-RD/transformer-deploy/blob/main/demo/quantization/quantization_end_to_end.ipynb) is related.

Just saw your repo has `Xenova/Bio_ClinicalBERT`.  I'll try those embeddings to see if they work for my use case.

**Comment by xenova** [comment]
It looks like you're trying to push to my transformers fork (not transformers.js): https://github.com/baloodevil/transformers

**Comment by xenova** [comment]
I've updated the docs with the correct tokenizer 👍 

> Working on converting to ONNX now. It seems to be stopped at Ignore MatMul due to non constant B: /[/distilbert/transformer/layer.5/attention/MatMul_1]

I'm converting it myself and this just takes a little long due to the large vocabulary size. On my computer it took around 15 minutes. For your convenience, I uploaded my version to the hub [here](https://huggingface.co/Xenova/ClinicalBERT), so feel free to use that instead of converting your own.
